quasar/main.tf

```diff
@@ -343,6 +343,107 @@ resource "aws_db_parameter_group" "quasar-qa-pg11" {
   }
 }
 
+resource "aws_db_parameter_group" "quasar-qa-pg12" {
+  name   = "quasar-qa-pg12"
+  family = "postgres12"
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
+  parameter {
+    name         = "rds.logical_replication"
+    value        = "1"
+    apply_method = "pending-reboot"
+  }
+
+  # Required for DMS CDC Replication:
+  # https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.PostgreSQL.html#CHAP_Source.PostgreSQL.v10
+  parameter {
+    name         = "max_logical_replication_workers"
+    value        = "500"
+    apply_method = "pending-reboot"
+  }
+
+  # Recommended by PGTuner tool: https://pgtune.leopard.in.ua/#/
+  # Sets effective RAM available to PG Query planner before using disk.
+  parameter {
+    name  = "effective_cache_size"
+    value = "12000000"
```

Comment:
It might be nice to re-write these to make units clearer (is it KB?) and how we got here... e.g:

```hcl
value = "${12 * 1000 * 1000}" # 12GB.
```

Reply:
Ooh, love this! Yah, will definitely change this!